src/Compilers/CSharp/Test/Syntax/Parsing/CollectionLiteralParsingTests.cs

```diff
@@ -5829,50 +5773,6 @@ public void InvokedCollectionLiteral2()
         EOF();
     }
 
-    [Fact]
-    public void AttributeOnStartOfLambda()
```

Comment:
this moved and test was renamed.  others in this same vein then were added after that.